src/messages/MClientReply.h

```diff
@@ -221,6 +229,9 @@ struct InodeStat {
         decode(fscrypt_auth, p);
         decode(fscrypt_file, p);
       }
+      if (struct_v >= 8) {
+        decode(optmetadata, p);
+      }
```

Comment:
@batrick now that the struct version is bumped up to 8, should't the decode macro at the top also reflect the bumped up version, i.e. `DECODE_START(8, p);`